core/timeline_api.php

```diff
@@ -57,13 +57,15 @@ function timeline_events( $p_start_time, $p_end_time, $p_max_events, $p_filter =
 	$t_result = history_query_result( $t_query_options );
 	$t_count = 0;
 
+	$t_previous_history_event = null;
```

Comment:
I don't see where this variable is used.

Reply:
Leftover of reverting event collapsing (3f3c024). Fixed.